documentation/source/user_section/tutorials/vertebral-labeling/extracting-specific-labels-for-registration.rst

```diff
@@ -1,5 +1,13 @@
 .. _extracting-specific-labels:
 
+..
+    comment:: In the 2024 SCT Course, this section was moved to the start of
+              the registration section to improve the flow of the course.
+              However, it is kept in the "labeling" section in the web
+              tutorials because each tutorial section is intended to be
+              self-contained. Having a discrepancy here might not be the best,
+              so we should reflect on whether we want to keep things this way.
```

Comment:
re: 4709a77a27dbe40b56c2a1d6f9edbae4df6103cb; I noticed the above discrepancy between the slides and the tutorials.

I didn't want to have to redo both the labeling and registration tutorials just to shift this one little step? So I wanted to keep the tutorials as-is. But, definitely worth discussion.

Reply:
I added a point to the next meeting agenda about this question.

Reply:
From meeting: it's ok!